dataprofiler/data_readers/data.py

```diff
@@ -65,7 +65,14 @@ def __new__(
             options = dict()
 
         if is_valid_url(input_file_path):
-            input_file_path = url_to_bytes(input_file_path, options)
+            if S3Helper.is_s3_uri(input_file_path, logger=logger):
+                storage_options = options.pop("storage_options", {})
```

Comment:
documentation updated on `dev-gh-pages` reflecting this now https://github.com/capitalone/DataProfiler/pull/1063/files